ui/js/models/resultsets_store.js

```diff
@@ -1023,7 +1023,8 @@ treeherder.factory('ThResultSetStore', [
                             });
                         });
                         platform.groups = _.sortBy(platform.groups, function(group){
-                            return jobGroupOrder[group.symbol];
+                            //remove all visual additions for Tier during sort
+                            return jobGroupOrder[group.symbol.split("[",1)];
```

Comment:
There should ideally be a single character of whitespace between the two params for the split method. You can test your js code during development to make sure it not just passes eslint in Travis, but also exceeds it.

eg.
http://old.jslint.com (paste your line above)
Missing space between ',' and '1'.